src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +8,18 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, Clementi Ave 3, #12-34, 231534";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in the format: BLOCK, STREET, UNIT, POSTAL_CODE";
+    public static final String ADDRESS_VALIDATION_REGEX = ".+,.+,.+,.+";
+    private static final String ADDRESS_SEPARATOR = ", ";
 
     public final String value;
```

Comment:
Keeping this variable (value) seems redundant and may cause future problems like inconsistent data as you have introduced new ways of handling the address value.